package/Dockerfile

```diff
@@ -2,31 +2,35 @@ FROM nginx:1.21.6-alpine
 
 LABEL maintainer "Rancher Labs <support@rancher.com>"
 ARG ARCH=amd64
-ENV DOCKER_URL_amd64=https://get.docker.com/builds/Linux/x86_64/docker-1.12.3.tgz \
-    DOCKER_URL_arm64=https://github.com/rancher/docker/releases/download/v1.12.3/docker-v1.12.3_arm64.tgz \
+ENV DOCKER_VERSION=23.0.3
+ENV DOCKER_URL_amd64=https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz \
+    DOCKER_URL_arm64=https://download.docker.com/linux/static/stable/aarch64/docker-${DOCKER_VERSION}.tgz \
     DOCKER_URL=DOCKER_URL_${ARCH}
-ENV CRIDOCKERD_URL=https://github.com/Mirantis/cri-dockerd/releases/download/v0.2.6/cri-dockerd-0.2.6.${ARCH}.tgz
+ENV CRIDOCKERD_VERSION=0.3.1
```

Comment:
Thanks for separating out the dependency versions, I think it'd be cleaner to put all env vars together on top of file - so they're easier to track/update in future. wdyt?

Reply:
Done. PTAL